en/installation/installation-of-centreon-ha/overview.md

```diff
@@ -0,0 +1,167 @@
+
+### Resources constraints
+
+Pacemaker offers various types of constraints:
+* Location: where the resource should or shouldn't run.
+* Colocation: how resources behave to each other.
+
+For example, Centreon-HA uses location constraints to specify to Pacemaker that the database process 
+needs to be up on backend nodes but not on frontend nodes. 
+
+Regarding colocation constraints, they can ensure a Virtual IP sticks to the master nodes and/or role. 
+Therefore Users, Pollers, and Daemons constantly interact with the primary node. 
+
+### QDevice and votes
+
+The configuration of a qdevice is **mandatory** to avoid split-brain and other situations nobody wants 
+to face in a Cluster. The server with the `quorum-device` role aims to obtain an absolute majority in a vote 
+to elect a master node or resource role.
+
+## Support
+
+### Softwares & operating systems 
+
+Centreon officialy supports Clustering on the following Products: 
+
+* Any Centreon Licensed Editions 
+* Centreon Map Server 
+
+And on the following Operating Systems: 
+
+* CentOS 7
+* RHEL 7
+
+The only official database server Centreon supports is MariaDB. 
+
+Nevertheless, note that we validated that the whole solution can run on MySQL 8 *with some 
+modifications*, only the [community](https://github.com/centreon-ha/issues) (or your DBAs) 
+can help and support you in running a Cluster on top of an Oracle MySQL server.
+
+For both MariaDB and Oracle MySQL, the replication configuration might be intrusive. We *strongly discourage you* setting 
+up a cluster on a server holding other application databases, we won't support it. 
+
+### Architectures
+
+Centreon supports both 2 and 4 nodes architectures. We recommend using a two nodes architecture except
+if your organization requires a systematic split of front and back servers or your monitoring 
+scope is above 5k Hosts. 
+
+Schemas below show both architecture flavor and network flows between servers. To get the complete network 
+flow matrix, refer to the architecture dedicated installation page.
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Two-nodes-cluster-->
+
+![image](../../assets/integrations/centreon-ha/centreon-ha-2-nodes-arch.png)
+
+Reach [this page](../../installation/installation-of-centreon-ha/installation-2-nodes.png) to start your two nodes setup! 
+
+<!--Four-nodes-cluster-->
+
+![image](../../assets/integrations/centreon-ha/centreon-ha-4-nodes-arch.png)
+
+Reach [this page](../../installation/installation-of-centreon-ha/installation-4-nodes.png) to start your two nodes setup!
+
+## Additionnal information
+
+### Server organization
+
+Setting up a Centreon-HA cluster might be overkill or at least not optimal when all your servers are running in 
+the same datacenter or the same rack. 
+
+In a perfect world, the primary and secondary nodes are running on different (geographical) sites, and the qdevice 
+communicate with both sites independently. 
```

Comment:
Do we need to be precise that all the cluster servers (device include) must be running on the same network?

Reply:
Ok, I add stg